### PR TITLE
Create RileyCz.md

### PR DESCRIPTION
**Pull request by RileyCz**
## Describe your project in 1-3 sentences

I am building a barometric altimeter which can either display a pressure altitude for a given barometric setting or display the current barometric pressure. 

## Hack Club Slack handle

Riley Czar

## Checklist before submitting proposal:

- [✅] I am a current high school student (or younger).
- [✅] I have submitted an RSVP at <https://hackclub.com/winter>
- [✅] I am a member of the Hack Club Slack (Join at <https://hackclub.com/slack>)
- [✅] I have self-reviewed my project proposal.

### CODE REVIEW

File: RileyCz.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+
```
Comment (exu3) [inline RileyCz.md:1]:
```suggestion
```

Remove empty line

File: RileyCz.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+| Adafruit BMP280 I2C | [https://www.adafruit.com/product/2651](https://www.adafruit.com/product/2651)  | $25.45 |
+| Touchscreen LCD Display | [https://www.amazon.ca/480x320-Screen-Module-Arduino-Without/dp/B07NWH47PV/](https://www.amazon.ca/480x320-Screen-Module-Arduino-Without/dp/B07NWH47PV/)  | $21.25 |
+| Battery Holder x2 | [https://www.pcboard.ca/cr2032-battery-holder-with-switch](https://www.pcboard.ca/cr2032-battery-holder-with-switch)  | $14.53 |
+| CR2023 Battery | [https://www.homedepot.ca/product/duracell-2032-lithium-coin-battery-with-bitter-coating-4-pack/1001619397](https://www.homedepot.ca/product/duracell-2032-lithium-coin-battery-with-bitter-coating-4-pack/1001619397)  | $11.96 |
+| Soldering Iron | [https://www.amazon.ca/Soldering-Adjustable-Temperature-Electric-Repairing/dp/B0BBZB3G26/](https://www.amazon.ca/Soldering-Adjustable-Temperature-Electric-Repairing/dp/B0BBZB3G26/)  | $10.99 |
+| Solder Lead Free | [https://www.digikey.ca/en/products/detail/american-hakko-products,-inc./611-1/6228764](https://www.digikey.ca/en/products/detail/american-hakko-products,-inc./611-1/6228764)  | $29.04 |
```
Comment (exu3) [inline RileyCz.md:27]: If this is your first time soldering, I might also recommend getting a solder iron stand as well as other useful supplies like a solder sucker, solder wick, flux, tip cleaner, etc. Sometimes you can find these tools in kits. "Soldering kit" as a search query on Amazon seems to have a number of results.

File: RileyCz.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+| Total           |                                       | $142.21 |
```
Comment (exu3) [inline RileyCz.md:28]: Do you need a breadboard & wires for prototyping? Do you need any passives (resistors, capacitors, diodes, etc) for your circuit?

**Review by exu3 — CHANGES_REQUESTED** [review]
Thanks for submitting a proposal @RileyCz!

The project idea looks great, but I think your budget might be missing a few things. I left some comments and would love to hear your thoughts. Once you're ready, feel free to request another review.

**Comment by RileyCz** [comment]
> Thanks for submitting a proposal @RileyCz!
> 
> The project idea looks great, but I think your budget might be missing a few things. I left some comments and would love to hear your thoughts. Once you're ready, feel free to request another review.

Thanks for the suggestions. I think a breadboard will be helpful for prototyping and I have updated it with some things I think I may need. If I missed anything please let me know.

**Review by exu3 — APPROVED** [review]
Looks great! The @hackclub/bank-ops team will follow up with next steps on receiving your grant through Hack Club Bank.

Excited to see what you ship!

**Review by MelSmith104 — COMMENTED** [review]
hey @RileyCz ! your project looks amazing! i love that you chose it because you want to be a pilot - that is so cool! im dm'ing you on slack right now so that we can chat about next steps for the grant!

**Review by MelSmith104 — APPROVED** [review]

**Pull request merged**